products/sentinel_one.py

```diff
@@ -326,7 +331,11 @@ def _get_dv_events(self, query_id: str) -> list[dict]:
 
     def process_search(self, tag: Tag, base_query: dict, query: str) -> None:
         build_query, from_date, to_date = self.build_query(base_query)
-        query = query + build_query
+        query = dict(map(lambda x: x.split(':'), query.split(',')))
```

Comment:
This is expected behavior since the `query` parameter only supports the target EDR's native query language so no formatting/parsing is required here.